docs/source/submitting_via_amulet.md

```diff
@@ -0,0 +1,96 @@
+# Submitting jobs to Singularity using Amulet
+PLEASE NOTE: Amulet is only intended for those with access to internal Microsoft compute resources. To access Amulet you
+must have an identity associated with the Microsoft tenant. This means you must be a Microsoft employee or approved
+external user.
+
+## Install Amulet
+This package is not included in the hi-ml environment definition, since these instructions only apply to users
+associated with the Microsoft tenant
+```bash
+$ pip install -U amlt --extra-index-url https://msrpypi.azurewebsites.net/stable/7e404de797f4e1eeca406c1739b00867
+```
+
+## Create an Azure ML Storage Account
+As stated in the [Amulet docs](https://amulet-docs.azurewebsites.net/main/setup.html#azure-storage-account), a Storage
+Account is required for storing information about your experiments, outputs of jobs etc. The Storage Account must be of
+type Storage V2. See the docs for steps on setting up.
+
+## Add your Singularity Workspace
+```bash
+$ amlt workspace add WORKSPACE_NAME --subscription SUBSCRIPTION_ID --resource-group RESOURCE_GROUP
+$ amlt workspace set-default VC_NAME WORKSPACE_NAME
+```
+
+## Create or checkout a project
+As stated in the docs, an [Amulet project](https://amulet-docs.azurewebsites.net/main/basics/00_create_project.html)
+"usually corresponds to a single research endeavor, e.g. a publication". Projects contain experiments which contain
+jobs. To create a project:
+```bash
+$ amlt project create <your-project-name> <storage-account-name>
+```
+To manage existing projects, use:
+```bash
+$ amlt project {list|checkout|remove}
+```
```

Comment:
We don't currently have a storage account per project, in the sense they mean, has this any consequence? what is shared in an amulet project?

Reply:
We would create 1 storage account for our team (or use an existing one) which would have a single container called 'amulet' (this is default) then each project would correspond to a folder within that